[Improvement] Add contributor docs, AI assistant rules, and fix helm-lint (#293)
mlrun/ce: .claude/skills/pr/SKILL.md

2. **Analyze the diff** carefully:
   - What components or templates were changed? (check which `templates/` subdirs, `values.yaml` sections, `requirements.yaml`, `Chart.yaml`)
   - Were any new values keys added? Do they need to be reflected in the three install-mode values files?
   - Were any Secrets, ConfigMaps, or port numbers changed? (potential breaking changes)
   - Was `Chart.yaml` version bumped? If not, flag it.
   - Were `requirements.yaml` or `requirements.lock` changed?
   - Does `charts/mlrun-ce/README.md` need updating (new NodePort, new component, new install step)?

3. **Detect breaking changes** — flag as breaking if any of:
   - A value key was renamed or removed
   - A Secret or ConfigMap name changed
   - A NodePort number changed
   - A sub-chart was upgraded with a major version bump
   - The storage credentials structure changed
   - Any hook annotation or hook-weight changed in a way that affects upgrade order

5. Provide an option PR title following the `[Scope] description` format, where Scope is one of: `['feature', 'fix', 'docs', 'improvement', 'revert', 'breaking', 'ci']`. For example: `[Feature] Add Redis support to mlrun-ce`.

> 6. **Fill the PR template** — produce the complete filled template below. Be specific and concrete; do not use placeholder text.

Reviewer: ordering

  - [ ] [User values Node Port](https://github.com/mlrun/ce/blob/development/charts/mlrun-ce/non_admin_installation_values.yaml)
  - [ ] [User values ClusterIP](https://github.com/mlrun/ce/blob/development/charts/mlrun-ce/non_admin_cluster_ip_installation_values.yaml)

---

### 🧪 Testing
<what was tested: lint, helm template dry-run, Kind cluster, manual — based on nature of changes>

---

### 🔗 References
- Ticket link:
- External links:
- Design docs links (Optional):

---

### 🚨 Breaking Changes?

- [ ] Yes (explain below)
- [ ] No

<if breaking: bullet list of what downstream consumers must change — value keys to rename, Secrets to recreate, ports to update>

---

### 🔍️ Additional Notes
<follow-up tasks, known issues, affected areas — omit if nothing to add>
```

Then replace each `[ ]` with `[x]` on items you can confirm from the diff, following the rules above.

After outputting the filled template, add a short **"Warnings"** section (outside the template) listing anything that needs human attention before opening the PR (missing version bump, unsynced values files, potential breaking changes, etc.).

> Between every sentence that end with a `.` add a two new lines to make it more readable.

Reviewer: ```suggestion
Between every sentence that ends with a `.`, add two new lines to make the output more readable.
```
